osbs/api.py

```diff
@@ -440,10 +440,11 @@ def _update_build_config_when_exist(self, build_json):
         return existing_bc
 
     @retry_on_conflict
-    def _update_build_config_with_triggers(self, build_json, triggers):
+    def _update_build_config_with_triggers(self, build_json, triggers, is_autorebuild="false"):
         existing_bc = self._get_existing_build_config(build_json)
         existing_bc['spec']['triggers'] = triggers
         build_config_name = existing_bc['metadata']['name']
+        existing_bc['metadata']['labels']['is_autorebuild'] = is_autorebuild
```

Comment:
Could we stay with boolean and only transform it here to str?

`... = "true" if is_autorebuild else "false" `

Reply:
sure

Reply:
changed